### ISSUE TITLE
🚀 [Forum] Redebeiträge aus dem BT

### ISSUE THREAD

**Issue by ulfgebhardt** [demokratie-live/democracy-client#638]
### [Forum] Redebeiträge aus dem BT
Redebeiträge aus dem Bundestag abbilden mit
- [ ] Video
- [ ] Prosa

**Comment by ManAnRuck** [comment]
@visionmakery Wenn ich das richtig erfasse, kommt es leider sehr häufiger dazu, dass bei einzelnen Reden über mehrere Anträge/Gesetze gleichzeitig debattiert wird. Hätte zur folge, dass die selben Videos in verschiedenen Prozeduren angezeigt wird. 
Der einzige Weg, für mich das zu differenzieren, ist anhand der Parteizugehörigkeit (Antragsteller <--> Redner).
Um das also relativ klar zu machen sehe ich da nur den Antragsteller ersichtlich zu machen, was ich jedoch nicht optimal finde (für themenbasierte Unvoreingenommenheit).

Hast du eine andere Idee?

**Comment by visionmakery** [comment]
Ja, das ist mir beim Eintragen von Bundestagsergebnissen auch (schon) aufgefallen. Oftmals steht am Ende einer Debatte die Abstimmung über ein (von der BReg eingebrachtes) Gesetz und zahlreiche Anträge (aus der Opposition). Da sich die Debatte dann auch auf mehrere Vorgänge bezieht, ist es zwar nicht optimal, aber unumgänglich sie in allen zugehörigen Vorgängen anzuzeigen, oder?

**Comment by visionmakery** [comment]
Sollten wir die Redebeiträge einführen, wird die Anzeige der Partei in den Detailinformationen notwendig

**Comment by ManAnRuck** [comment]
link zum video
replace ###VIDEOID###

`https://static.p.core.cdn.streamfarm.net/1000153copo/ondemand/145293313/###VIDEOID###/###VIDEOID###_h264_1920_1080_5000kb_baseline_de_5000.mp4`

**Comment by ManAnRuck** [comment]
Aktuelle Live stream url:
https://cldf-hlsgw.r53.cdn.tv1.eu/1000153copo/live/app144277506/BTTV_1/QdoHEQIoadKSHi6oKEp5ziZ2q/J7ekv9+hgpBxTSy2G2KefdbYTKWe5R2gMXKyP0.m3u8

immer die gleiche?

**Comment by hermann-san** [comment]
ich muss sagen das mich Bundestagsreden nicht viel schlauer machen. Für mich ist das in etwa so als würde ich Werbung im TV anschauen. 
Wenn ich wissen will ob ich für oder gegen ein Gesetz bin dann lasse ich mich üblicherweise lieber von dem journalistischen Format meines Vertrauens informieren. Am besten sogar noch von verschiedenen Medien (Zeitung und/oder TV und/oder  Online).
Ich lese z.B. meine regionale Zeitung (wenn ich eine gute hätte) und schau was die über das Gesetz sagen. Dann schau ich vielleicht noch in eine überregionale Zeitung und in einen Online Blog. So mache ich mir ein Bild über ein Gesetz. Ich schaue mir selten die Werbeshow der Politiker an und versuche das Kauderwelsch aus Buzzwords und Schönfärberei, etc zu dechiffrieren. Dafür bezahle ich Journalisten die sich ausführlichst die Zeit nehmen können um das Gesetz komplett zu lesen, Experten zu befragen und mir die Zusammenfassung zugeben. 
Ich denke diese Vorgehen und die Erwartungshaltung an Journalismus haben viele und wie ich denke zu recht.  Ob das  im heutigen Journalismus gut funktioniert ist wieder eine ganz andere Frage ;-)  , deswegen kann man sich ja aussuchen woher man seine Informationen bezieht. Jedenfalls schaue ich mir nicht aktiv Bundestagsreden an um zu erfahren was in einem Gesetz steht und was es bedeutet. BT-Reden sind für mich eher Beiwerk.
Deswegen fände ich es sinnvoller und wichtiger Online-Artikel von verifizierten Medien zu verlinken (Pro oder Contra Gesetz). Nähere Details zu dieser Idee hab ich [in einem anderen Issue geschrieben](https://github.com/demokratie-live/democracy-client/issues/1543#issuecomment-1118025669)
